frank-doc-doclet/pom.xml

```diff
@@ -238,21 +242,22 @@
 		<dependency>
 			<groupId>junit</groupId>
 			<artifactId>junit</artifactId>
-			<version>4.13</version>
+			<version>4.13.2</version>
 			<scope>test</scope>
 		</dependency>
 
 		<dependency>
 			<groupId>org.projectlombok</groupId>
 			<artifactId>lombok</artifactId>
-			<version>1.18.22</version>
+			<version>1.18.30</version>
 			<scope>provided</scope>
 		</dependency>
 		<dependency>
 			<groupId>com.sun</groupId>
 			<artifactId>tools</artifactId>
 			<version>1.8</version>
-			<scope>provided</scope>
+			<scope>system</scope>
+			<systemPath>${java.home}/../lib/tools.jar</systemPath>
```

Comment:
Ik weet niet of dit op de CI goed gaat.. de dependency staat bij ons op de nexus. Dus dit is niet nodig.

Reply:
`java.home` is by default gedefinieerd door de JDK. De `tools.jar` werkt bij mij niet by default, met Java 8. Dus dit lijkt me een meer standaard oplossing. Maar goed, het is even een tussentijdse fix, want in mijn nieuwe branch gaat die hele dep er uit.